include/dpp/unicode_emoji.h

```diff
@@ -3,9 +3,11 @@
 namespace dpp {
 
 /**
- * The unicode emojis in this namespace are auto-generated from https://raw.githubusercontent.com/ArkinSolomon/discord-emoji-converter/master/emojis.json
+ * @brief Emoji unicodes.
  *
- * If you want to use this, you have to pull the header in separately. e.g.
+ * @note The unicode emojis in this namespace are auto-generated from https://raw.githubusercontent.com/ArkinSolomon/discord-emoji-converter/master/emojis.json
```

Comment:
you edited the note, telling you not to hand edit the file 😁 
if you want to change this file you have to change the php file that generates it under buildtools. be sure to test it!

Reply:
Ahhh I thought the php was just editing the content below 😅